### PR TITLE
Load KAKAROT_ADDRESS and PROXY_CLASS_HASH from .env

### PR DESCRIPTION
**Pull request by ClementWalter**
Time spent on this PR: 0.5

# Pull Request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Testing
- [ ] Other (please describe):

# What is the current behavior?

Kakarot contract address and proxy account class hash are stored in a `constants.rs` file however they are meant to change often (depending on devnet/testnet/mainnet, on new deployments)

# What is the new behavior?

Values loaded from `.env` file

# Does this introduce a breaking change?

- [ ] Yes
- [x] No

# Other information

### CODE REVIEW

**Comment by Eikix** [comment]
Tests fail, could it be that in CI, env variables aren't properly injected?

File: rpc/tests/utils.rs [hunk 1]
```diff
@@ -29,7 +30,12 @@ use kakarot_rpc_core::{client::KakarotClientImpl, utils::wiremock_utils::setup_w
 pub async fn setup_kakarot_eth_rpc() -> KakarotEthRpc {
     let starknet_rpc = setup_wiremock().await;
 
-    let kakarot_client = KakarotClientImpl::new(&starknet_rpc).unwrap();
+    let kakarot_client = KakarotClientImpl::new(
+        &starknet_rpc,
+        FieldElement::from_dec_str("0").unwrap(),
```
Comment (Eikix) [inline rpc/tests/utils.rs:35]: either these env vars are somehow not valid, either the devnet URL inside our test suite is not working

**Review by Eikix — COMMENTED** [review]
v nice PR! let's merge once tests pass

File: rpc/tests/utils.rs (hunk 1 above)
Comment (ClementWalter) [reply rpc/tests/utils.rs]: I used the one I removed in the tests; I think that these values are required for being consistent with the other hard coded fixtures (the json files)

File: core/src/client/mod.rs [hunk 2]
```diff
@@ -812,7 +825,7 @@ impl KakarotClient for KakarotClientImpl {
 
         let request = FunctionCall {
             // This FieldElement::from_dec_str cannot fail as the value is a constant
```
Comment (Eikix) [inline core/src/client/mod.rs:827]: `This FieldElement::from_hex_be cannot fail as the value is a constant`

**Review by Eikix — DISMISSED** [review]
lgtm apart from the comment typo

**Pull request merged**